src/common/tests/btree.c

```diff
@@ -830,8 +830,10 @@ ik_btr_drain(void **state)
 	int		 i;
 
 	D_ALLOC_ARRAY(arr, drain_keys);
-	if (arr == NULL)
+	if (arr == NULL) {
 		fail_msg("Array allocation failed");
+		fail();
```

Comment:
doesn't fail_msg abort the test?

Reply:
indeed, I guess we need to call return explicitly to make coverity happy